projects/components/src/table/data/table-cdk-column-util.ts

```diff
@@ -17,5 +17,11 @@ export namespace TableCdkColumnUtil {
   export const unsortOtherColumns = (sortedColumn: TableColumnConfig, otherColumns: TableColumnConfig[]): void =>
     otherColumns
       .filter(column => column.field !== sortedColumn.field)
-      .forEach(filteredColumn => (filteredColumn.sort = undefined));
+      .forEach(filteredColumn => {
+        filteredColumn.sort = undefined;
+        filteredColumn.sortable = false;
```

Comment:
Should we set `sortable = false`? Wouldn't this take out the sort feature of an existing column?

Reply:
Shit. Yes. Nice catch!